mw-webapp/cypress/e2e/allWays.cy.ts

```diff
@@ -8,6 +8,24 @@ describe('All Ways scope tests', () => {
 
   beforeEach(() => {
     cy.visit('/');
+    headerSelectors.getBurgerMenu().click();
+    navigationMenuSelectors.menuItemLinks.getAllWaysItemLink().click();
   });
 
+  afterEach(() => {
+    cy.clearAllStorage();
+  });
+
+  it('NoAuth_AllWays_SelectTableView', () => {
+    const tableHeadersEn = Object.values(allWaysPageContent.waysTable.columns).map(column => column.en);
+    tableHeadersEn.push(allWaysPageData.starSimbol);
+
+    allWaysSelectors.filterViewBlock.getTableViewButton().click();
+
+    allWaysSelectors.allWaysTable.getTableTr().find('th').each((el, index) => {
```

Comment:
I see you added data attribute for tr, lets add to to th too

Reply:
done